Ginger/GingerCoreNET/Application Models/Delta/APIDelta/APIDeltaUtils.cs

```diff
@@ -0,0 +1,107 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GingerCoreNET.Application_Models
+{
+    public class APIDeltaUtils
+    {
+        public static void ComparisonUtility(ObservableList<ApplicationAPIModel> learnedAPIModelsList, ObservableList<DeltaAPIModel> deltaAPIModelsList)
```

Comment:
1- call method "DoAPIModelsCompare"
2- it should return the DeltaAPIModels list as output instead getting it as input

Reply:
Done !